### ISSUE TITLE
Feature importance metrics compared to sklearn

### ISSUE THREAD

**Issue by BrunoBarre** [dmlc/xgboost#4466]
Hello and thanks for the implementation :+1: 

In Sklearn, for example with DecisionTreeClassifier, we have the feature_importances_, which returns the (normalized) Gini Importance. For the criterion we can choose between the Gini Impurity and the Information Gain.

In the XGBClassifier, the parameter importance_type can be set to "gain" and "total_gain", among others. 

I would like to, kind of, be able to compare the two feature importance rankings. So my first question is, which metrics is used in XGBoost (I couldn't find it in the docs), is it Information Gain?

Then, if I want to compare to Sklearn, should I select "total_gain" (and not "gain") ?

Thanks!

**Comment by hcho3** [comment]
XGBoost has no feature importance measure that's equivalent to scikit-learn's. The reason is that scikit-learn uses Gini Impurity and Information Gain to choose optimal splits, whereas XGBoost uses gradient statistics.
